QAOA/src/Examples.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+
+namespace Quantum.QAOA
+{
+    using Microsoft.Quantum.QAOA.QaoaHybrid;
+    using System;
+
+
+    class Examples
+    {
+        static void Main(string[] args)
+        {
+            //PARAMETERS
+            int numberOfIterations = 50;
+            int p = 3;
+            int numberOfRandomStartingPoints = 1;
```

Comment:
In general, we tend to use `var` in C# code. That can't be used with array initializers like at line 21, but it's great for initializing scalars.
```suggestion
            var numberOfIterations = 50;
            var p = 3;
            var numberOfRandomStartingPoints = 1;
```